id_test.js

```diff
@@ -34,6 +34,9 @@ Sum.prototype[equals] = function(x) {
     return this.v.equals ? this.v.equals(x.v) : this.v === x.v;
 };
 
+// Special function for monad identity laws.
+const f = x => Id.of(x + x);
+
```

Comment:
Without it, `x => Id.of(x)` is simply the same as `Id.of`. In that case, there is no difference between the `leftIdentity` and `rightIdentity` test cases. So I opted for a simple non-trivial monad-returning function.